search-app/src/cmr/search/services/parameters/conversion.clj

```diff
@@ -197,47 +197,14 @@
    :concept-id :string
    :keyword :keyword})
 
-;; TODO: Generic work: See if we can wrap this in a doseq for all generic types or get from 
-;; a config file.
-(defmethod common-params/param-mappings :dataqualitysummary
-  [_]
-  {:name :string
-   :id :string
-   :provider :string
-   :native-id :string
-   :concept-id :string})
-
-(defmethod common-params/param-mappings :orderoption
-  [_]
-  {:name :string
-   :id :string
-   :provider :string
-   :native-id :string
-   :concept-id :string})
-
-(defmethod common-params/param-mappings :serviceoption
-  [_]
-  {:name :string
-   :id :string
-   :provider :string
-   :native-id :string
-   :concept-id :string})
-
-(defmethod common-params/param-mappings :serviceentry
-  [_]
-  {:name :string
-   :id :string
-   :provider :string
-   :native-id :string
-   :concept-id :string})
-
-(defmethod common-params/param-mappings :grid
-  [_]
-  {:name :string
-   :id :string
-   :provider :string
-   :native-id :string
-   :concept-id :string})
+(doseq [concept-type (cc/get-generic-concept-types-array)]
```

Comment:
so in the future I assume we will append a function here which will read the config file and add new params? Has the TODO been resolved?

Reply:
That's a good question. I deleted the original TODO because it read "use doseq OR get from config file" and so it was technically resolve ;) But I think the remaining question of, "should we move this into a config file?" is a good one.

Reply:
We discussed this and decided to put the TODO back in for adding additional params through a config file, and create a ticket for it.